tests/functional/api/test_query.py

```diff
@@ -31,6 +31,12 @@ def test_basic_query(eth_tester_provider):
     ]
 
 
+def test_relative_block_query(eth_tester_provider):
+    chain.mine(10)
+    df = chain.blocks.query("*", start_block=-8, stop_block=-2)
+    assert len(df) == 7
```

Comment:
might be nice to check we have the right block numbers, at least at the start and stop blocks

Reply:
Thanks for this, think I identified an off-by-one bug

Reply:
nvm, false alarm. `-1` means "last block"